omod-common/src/main/java/org/openmrs/module/webservices/helper/TaskServiceWrapper.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.webservices.helper;
+
+import org.openmrs.scheduler.TaskDefinition;
+import org.openmrs.scheduler.SchedulerService;
+import org.openmrs.scheduler.SchedulerException;
+import org.openmrs.api.context.Context;
+
+import java.util.Collection;
+
+public class TaskServiceWrapper {
+	
+	public TaskDefinition getTaskById(String id) {
+		return Context.getSchedulerService().getTask(Integer.parseInt(id));
+	}
+	
+	public Collection<TaskDefinition> getScheduledTasks() {
+		return Context.getSchedulerService().getScheduledTasks();
+	}
+	
+	public Collection<TaskDefinition> getRegisteredTasks() {
+		return Context.getSchedulerService().getRegisteredTasks();
+	}
+	
+	public boolean isTaskStarted(TaskDefinition task) {
+		return task.getStarted();
```

Comment:
I do not see value in a method like this. Why doesn't the caller just do task.getStarted()?

Reply:
I wished to use TaskServiceWrapper for all kind of works. So Implement all the functionalities through TaskServiceWrapper. 
Is it bad, then I will remove :-)

Reply:
Generally, more methods is more code to maintain. So let us not write any extra code unless there is real value that it adds. Just wrapping a method call in another method call does not seem to add any value. Does it? 😊

Reply:
Yah absolutely. It will be just faking the Code :blush: 
I will do required changes and remove unwanted methods here